components/library/libabw/Makefile

```diff
@@ -23,16 +23,17 @@
 BUILD_BITS=32_and_64
 include ../../../make-rules/shared-macros.mk
```

Comment:
/usr/bin/pkgdepend generate -m -d /export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386/mangled -d /export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386 -d /export/home/andreas/oi-userland/components/library/libabw/build -d /export/home/andreas/oi-userland/components/library/libabw -d libabw-0.1.3 /export/home/andreas/oi-userland/components/library/libabw/build/manifest-i386-libabw.mangled >/export/home/andreas/oi-userland/components/library/libabw/build/manifest-i386-libabw.depend
Couldn't find 'usr/share/doc/libabw/html/annotated.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libabw
	/export/home/andreas/oi-userland/components/library/libabw/build
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libabw/libabw-0.1.3
Couldn't find 'usr/share/doc/libabw/html/classPerfect__Hash-members.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libabw
	/export/home/andreas/oi-userland/components/library/libabw/build
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libabw/libabw-0.1.3
Couldn't find 'usr/share/doc/libabw/html/classPerfect__Hash.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libabw
	/export/home/andreas/oi-userland/components/library/libabw/build
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libabw/libabw-0.1.3
Couldn't find 'usr/share/doc/libabw/html/classes.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libabw
	/export/home/andreas/oi-userland/components/library/libabw/build
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libabw/libabw-0.1.3
Couldn't find 'usr/share/doc/libabw/html/functions.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libabw
	/export/home/andreas/oi-userland/components/library/libabw/build
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libabw/libabw-0.1.3
Couldn't find 'usr/share/doc/libabw/html/functions_func.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libabw
	/export/home/andreas/oi-userland/components/library/libabw/build
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libabw/libabw-0.1.3
Couldn't find 'usr/share/doc/libabw/html/functions_vars.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libabw
	/export/home/andreas/oi-userland/components/library/libabw/build
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libabw/libabw-0.1.3
Couldn't find 'usr/share/doc/libabw/html/globals_defs.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libabw
	/export/home/andreas/oi-userland/components/library/libabw/build
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libabw/libabw-0.1.3
Couldn't find 'usr/share/doc/libabw/html/structxmltoken-members.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libabw
	/export/home/andreas/oi-userland/components/library/libabw/build
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libabw/libabw-0.1.3
Couldn't find 'usr/share/doc/libabw/html/structxmltoken.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libabw
	/export/home/andreas/oi-userland/components/library/libabw/build
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libabw/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libabw/libabw-0.1.3
gmake: *** [/export/home/andreas/oi-userland/make-rules/ips.mk:498: /export/home/andreas/oi-userland/components/library/libabw/build/manifest-i386-libabw.depend] Error 1

Reply:
I will just drop the documentation in this case...